### DISCUSSION TITLE
Ability to disable 3D mode for a profile

### DISCUSSION THREAD

**Discussion by kostrse** [osmandapp/OsmAnd#21032] [Ideas]
Most of the time I use topographic maps in 2D mode and very frequently, due to touch manipulations, the screen does unintentionally transition to the 3D mode, which is not desirable in some situations.
Please add an option switch in profile settings to optionally disable 3D mode for a given profile.

Thanks!

**Comment by kostrse** [comment]
It almost always happen when I try to rotate the map using two fingers, this is very annoying.

**Comment by ThureHauser** [comment]
I agree with an option to turn off 3D-tilted view for profiles and the entire app. 
It makes no sense when using NorthUp, as I prefer

**Comment by readonly24** [comment]
Maybe an option here?

<table style="width:100%">
  <tr>
<th><img src="https://github.com/user-attachments/assets/ca89709f-ab9f-4f1b-bddf-71f154111e76"  width="300" height="auto"/> </td>
  </tr>
</table>

**Comment by pokotiX** [comment]
I have the same wish. It would be really convenient if there was a way to permanently turn off 3D.

**Comment by mmoser18** [comment]
I second this wish! This constant unintended kicking-in of 3D mode is driving me nuts! Why can't one disable this BS?
